config/routes.rb

```diff
@@ -1,3 +1,9 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 end
```

Comment:
- [ ] [**_Optional_**] Good job implementing only 4 of the required paths. But I just wanted to mention that using rails `resources` DSL by defaults adds named path helpers that you can later use to refer to different routes easily. See the difference below:

![Screenshot from 2021-12-11 09-49-44](https://user-images.githubusercontent.com/61377322/145668211-027fe460-8281-490d-b110-e904c3b04a45.jpg)

Reply:
Thanks for the comment